docs/modules/ROOT/nav.adoc

```diff
@@ -1,36 +1,40 @@
 * xref:index.adoc[Overview]
-//* xref:wizard.adoc[Wizard]
-* xref:usage.adoc[Usage]
```

Comment:
Small thing, but should we change `index.adoc` to `overview.adoc`?

Reply:
I don't think so, it's the actual landing page. It's also consistent with other docsites (e.g. Contracts for Solidity)